tools/urls-check

```diff
@@ -0,0 +1,122 @@
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+
+import os
+from os.path import dirname, abspath
+import sys
+import time
+import argparse
+import subprocess
+import urllib
+from urllib.request import urlopen, Request
+
+sys.path.append(os.path.join(dirname(dirname(abspath(__file__))), "bots"))
+import task
+
+sys.dont_write_bytecode = True
+
+DAYS = 3
+TASK_NAME = "Validate all URLs"
+
+def main():
+    parser = argparse.ArgumentParser(description=TASK_NAME)
+    parser.add_argument("-v", "--verbose", action="store_true", help="Verbose output")
+    opts = parser.parse_args()
+
+    task.verbose = opts.verbose
+
+    since = time.time() - (DAYS * 86400)
+
+    # When there is opened issue, don't do anything
+    issues = task.api.issues(state="open")
+    issues = [i for i in issues if i["title"] == TASK_NAME]
+    if issues:
+        return
+
+    issues = task.api.issues(state="closed", since=since)
+    issues = [i for i in issues if i["title"] == TASK_NAME]
+
+    # If related issue was not modified in last n-DAYS, then do your thing
+    if not issues:
+        (success, err) = check_urls(opts.verbose)
+        if err:
+            # Create a new issue
+            data = {
+                "title": TASK_NAME,
+                "body": err,
+                "labels": ["bot"]
+            }
+            task.api.post("issues", data)
+        else:
+            # Try to comment on the last issue (look in the last 4*DAYS)
+            # If there is not issue to be found, then just open a new one and close it
+            success = success or "Task haven't produced any output"
+            since = time.time() - (DAYS * 4 * 86400)
+            issues = task.api.issues(state="closed", since=since)
+            issues = [i for i in issues if i["title"] == TASK_NAME]
+            if issues:
+                task.comment(issues[0], success)
+            else:
+                data = {
+                    "title": TASK_NAME,
+                    "body": success,
+                    "labels": ["bot"]
+                }
+                new_issue = task.api.post("issues", data)
+                task.api.post("issues/{0}".format(new_issue["number"]), {"state": "closed"})
+
+def check_urls(verbose):
+    command = r'git grep -IEho "(https?)://[-a-zA-Z0-9()@:%_\+.~#?&=/]+" pkg | sort -u'
+    urls = subprocess.check_output(command, shell=True, universal_newlines=True).split("\n")
+    failed = []
+    for url in urls:
+        if not url:
+            continue
+
+        if "example.com" in url: # Some tests use demo urls
+            continue
+
+        if "sodipodi.sourceforge.net" in url: # Some .svg files contain url that does not exist
+            continue
+
+        if url.endswith(")"): # The regex is not ideal and sometimes there is trailing ')'
+            url = url[:-1]
```

Comment:
`url = url.strip(')')`

However, it seems unusual to have `(` and `)` in URLs in the first place. We don't have any in our repo. This is also not enough, as we still get this hit:

    https://github.com/abrt/reportd/issues/8).

So I suggest to simply toss `()` out of the RE, and drop this stripping.